.editorconfig

```diff
@@ -0,0 +1,47 @@
+[*]
```

Comment:
The idea is these preferences are what are needed for rubucop / code climate to be happy such as trailing newlines, certain number of spaces for indentation etc...

I can remove if you want though ?

Reply:
Rubocop and code climate will ensure our code styles will be applied anyway. I can see though that this file can be helpful too. How about we make it optional to avoid people being confused when seeing unexpected for them settings in their editor? Maybe we could rename it to something like `.editorconfig.example` and make a note in the readme for those who want to use it?

Reply:
Can do - the idea is though that the errors that waste time (the famous stupid CR at end of file for example) do not arise in the first place so makes sense to force everyone to use those particular settings which is the idea of the .editorconfig

Reply:
I believe it is acceptable and totally normal to have linters to secure that our style guides will be respected. However, injecting settings which will be auto-loaded on dev's local environment is wrong, can mess up the local environments, create frustration and can raise security concerns.

Imagine someone changes the `.editorconfig`. Everyone who will pull the changes will be affected with those wrong editor settings. I would teat that as a security breach as my editor will auto-load those changes.

Finally, for example the CRLF/LF is one of the most common issues but can't be totally controlled by the text editor as Git can also force it back after the dev will commit changes.

Reply:
Fair enough - ive removed it now and just done   a .example version